### PR TITLE
Add a skeleton for Datawind's Pocket Surfer (pocket internet terminal)

### PR DESCRIPTION
**Pull request by clawgrip**
New not working machine
---------------------------------
Pocket Surfer (prototype) [Chan Chu Yo]

### CODE REVIEW

File: src/mame/skeleton/pcksurfer.cpp [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+
+    TODO:
+    - Everything
+*/
+
+#include "emu.h"
+
+#include "cpu/arm7/arm7.h"
+#include "cpu/arm7/arm7core.h"
```
Comment (cuavas) [inline src/mame/skeleton/pcksurfer.cpp:44]: You shouldn’t need to include internal headers.

File: src/mame/skeleton/pcksurfer.cpp [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	ROM_LOAD( "lh79524.bootrom.bin", 0x0000, 0x2000, NO_DUMP )
+
+	ROM_REGION( 0x100, "keyboard", 0 )
+	ROM_LOAD( "attiny28l.bin", 0x000, 0x100, NO_DUMP ) // 2K Bytes of Flash
+ROM_END
+
+} // anonymous namespace
+
+//    YEAR  NAME       PARENT COMPAT MACHINE    INPUT      CLASS            INIT        COMPANY     FULLNAME                     FLAGS
+COMP( 2006, pcksurfer, 0,     0,     pcksurfer, pcksurfer, pcksurfer_state, empty_init, "Datawind", "Pocket Surfer (prototype)", MACHINE_IS_SKELETON )
```
Comment (cuavas) [inline src/mame/skeleton/pcksurfer.cpp:91]: Missing final newline.

File: src/mame/skeleton/pcksurfer.cpp [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+|     |SBTM1  |   |            |                                |      |       |
+|     |Bluetooth  |   SIM      |                                |______|       |
+|     |_______|   |  SOCKET    | ___         _____________            _______  |
+|   ___________   |   AND      ||___|  ___   | IS42S16400B|           |      | |
+|   |U4 Flash  |  |  COMMS     |       |  |  | DRAM       | ________  |IS42S16400B
+|   |          |  |            | LVX457->_|  |____________| |SHARP  | |DRAM  | |
+|   |__________|  |____________|       ___                  |LH79524| |      | |
+|                           ___        |  |                 |NOF    | |      | |
+|                    LCX125->__|       |_<-LCX125           |_______| |______| |
+|______________________________________________________________________________|
```
Comment (cuavas) [inline src/mame/skeleton/pcksurfer.cpp:32]: Do you know what cellular modem it uses?

File: src/mame/skeleton/pcksurfer.cpp (hunk 1 above)
Comment (clawgrip) [reply src/mame/skeleton/pcksurfer.cpp]: Done, thanks!!

File: src/mame/skeleton/pcksurfer.cpp (hunk 2 above)
Comment (clawgrip) [reply src/mame/skeleton/pcksurfer.cpp]: Done, thanks!!

File: src/mame/skeleton/pcksurfer.cpp (hunk 3 above)
Comment (clawgrip) [reply src/mame/skeleton/pcksurfer.cpp]: Yes, SIM300 v7.03. I've added a note.

Thanks!

**Pull request merged**